lld/wasm/Driver.cpp

```diff
@@ -556,6 +554,11 @@ static void readConfigs(opt::InputArgList &args) {
   config->zStackSize =
       args::getZOptionValue(args, OPT_z, "stack-size", WasmPageSize);
 
+  // -Bdynamic by default if -pie or -shared is specified.
+  if (config->pie || config->shared) {
+    config->isStatic = false;
+  }
```

Comment:
The convention removes braces in this single line statement case.

Reply:
fixed. thank you.